tests/utils.js

```diff
@@ -0,0 +1,18 @@
+/* global describe it expect */
+
+const expect = require('chai').expect;
+
+describe('Utility functions', () => {
+  describe('The `makeTempDir` utility', () => {
+    it('Should return an object with `tempDir` and `tempDirCreate` entries')
```

Comment:
Keep in mind this AirBnB rule as you implement this.
https://github.com/airbnb/javascript#objects--prototype-builtins